recipes-core/systemd/systemd_244.5.bbappend

```diff
@@ -0,0 +1,5 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
```

Comment:
In which version this patch is applied? Probably we need to apply it on all 244? systemd_244.%.bbaapend?

Reply:
fix fas added to 249 version